lib/ocpp/v201/charge_point.cpp

```diff
@@ -2113,7 +2113,7 @@ void ChargePoint::notify_report_req(const int request_id, const std::vector<Repo
     }
 }
 
-AuthorizeResponse ChargePoint::authorize_req(const IdToken id_token, const std::optional<CiString<5500>>& certificate,
+AuthorizeResponse ChargePoint::authorize_req(const IdToken id_token, const std::optional<CiString<10000>>& certificate,
```

Comment:
TODO: find out implications for backwards compatibility with 2.0.1

Reply:
accept